modules/raft/src/main/java/org/apache/ignite/raft/jraft/core/FSMCallerImpl.java

```diff
@@ -603,6 +603,8 @@ private void doSnapshotSave(final SaveSnapshotClosure done) {
         }
 
         SnapshotMetaBuilder metaBuilder = msgFactory.snapshotMeta()
+            .cfgIndex(confEntry.getId().getIndex())
+            .cfgTerm(confEntry.getId().getTerm())
```

Comment:
These 2 fields are only needed for partitions, and for partitions we don't save snapshot files. Hence, a question: do we really need to fill them here?

Reply:
discussed in private